tutorials/getting_started/getting_started.md

```diff
@@ -0,0 +1,181 @@
+# Getting Started
+
+Welcome to AllenNLP!
+
+## Installing using Docker
+
+The easiest way to get started is using Docker. Assuming you have Docker installed, just run
+
+```bash
+docker run -p 8000:8000 -it --rm allennlp/allennlp-cpu
+```
+
+(If your machine has GPUs, use `allenlp-gpu` instead.)
+
+This will download the latest `allennlp` image to your machine
+(unless you already have it),
+start a Docker container, and launch an interactive shell.
+(It also exposes port 8000, which is where the demo server runs,
+ and shuts down the container when you exit the interactive shell.)
+
+## Installing Not Using Docker
```

Comment:
For the "Getting Started" tutorial I'd like to keep the friction as low as possible. I'd hate for people not to try out AllenNLP because they don't want to install Docker.